package/MDAnalysis/coordinates/TRJ.py

```diff
@@ -1130,6 +1228,11 @@ class NCDFPicklable(scipy.io.netcdf.netcdf_file):
     mmap : None or bool, optional
         Whether to mmap `filename` when reading. True when `filename`
         is a file name, False when `filename` is a file-like object.
+    version : {1, 2}, optional
+        Sets the netcdf file version to read / write. 1 is classic, 2 is
+        65-bit offset format. Default is 1.
+    maskandscale : bool, optional
```

Comment:
Why, what? Links ... and when would one want to use it?

Are `version` and `maskandscale` options that users should play with?

Reply:
So this class essentially is just a wrapper around https://docs.scipy.org/doc/scipy/reference/generated/scipy.io.netcdf_file.html with a couple of extra methods to make it pickle-able. I just added the extra options because a) maskandscale needed to be propagated (although it's False by default currently I wanted to make sure we could be explicit here in case it changed in the future to match netCDF4's True-by-default), b) there might be some obscure use case where someone will want to use this for something else and will want the ability to use the other parameters.

Rather than detailing things too much (and also at the risk of just ending up copying the scipy docs verbatim), should we just link to https://docs.scipy.org/doc/scipy/reference/generated/scipy.io.netcdf_file.html ?

Reply:
Link, but also say that for normal ncdf behavior it should be left at the default